perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -226,6 +250,28 @@ def ApiBasedBenchmarks(results, metadata, vm, storage, test_script_path,
       # User only wants to run the CLI based tests, do nothing here:
       return
 
+    def BuildCommandString(args):
+      """Build a command string for the API test script.
+
+      Args:
+        A list of strings. These will become space-separated arguments to the
+          test script.
+
+      Returns:
+        A string that can be passed to vm.RemoteCommand.
+      """
+
+      cmd_parts = [
+          test_script_path,
+          '--storage_provider=%s' % storage
```

Comment:
Yes. If we decide to move the function outside of ApiBasedBenchmarks(), then test_script_path, storage, azure_command_suffix, and host_to_connect will become parameters.